api/types/filters/parse.go

```diff
@@ -230,6 +230,24 @@ func (filters Args) Validate(accepted map[string]bool) error {
 	return nil
 }
 
+// ValidateBoolValue ensures that the field filter are valid boolean value.
+// It returns an error as soon as it finds an invalid field value.
+func (filters Args) ValidateBoolValue(field string) (bool, error) {
```

Comment:
Hi @thaJeztah The issue is that that function
 support 1, t, T, TRUE, true, True, 

Docker CLI/API support  just true and 1.

Wdyt ?